packages/terra-theme-context/src/terra-dev-site/doc/ThemeContext/ThemeContext.1.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import { Badge } from 'terra-theme-context/package.json?dev-site-package';
+
+import ThemeContextProviderExample from '../example/ThemeContextProviderExample?dev-site-example';
+import ThemedComponent from '../example/ThemedComponent?dev-site-example';
+
+<Badge />
+
+# Terra Theme Context
+A react context to provide the current theme. This package is intended to be used by terra components only, Any terra consumers or custom terra components should consume the context from the terra-application package.
+
+This context can be used to apply any theme related changes to a component.
+
+The most common use would be to apply a theme class to the root tag of the component to apply theme variables. See below for an example.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-theme-context`
+
+## Usage
+
+```jsx
+import ThemeContext from 'terra-theme-context';
+```
+
+## Component Features
+
+ * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+ * [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+ * [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+ * [Internationalization Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#internationalization-i18n-support)
+ * [Localization Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#internationalization-i18n-support)
+ * [LTR/RTL Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#ltr--rtl-support)
```

Comment:
I"m not sure any of these features apply since this component only adds a context

Reply:
Fair